tests/app/questionnaire/test_router.py

```diff
@@ -189,7 +186,31 @@ def test_return_to_summary_next_location_url(self):
             "/questionnaire/sections/property-details-section/", next_location
         )
 
-    def test_return_to_final_summary_next_location_url(self):
+    def test_linear_questionnaire_get_next_location_url_routes_to_submit_page_when_questionnaire_completed(
```

Comment:
I can't see what this is testing that is different to `test_last_block_no_section_summary_next_location_url_is_submit_page`?

Reply:
Have been refactored.